charts/turing/templates/turing-service-account.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if or .Values.serviceAccount.create .Values.clusterConfig.useInClusterConfig -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "turing.serviceAccountName" . }}
```

Comment:
This could be "default" in some cases. Do you think we should still apply this resource?

Reply:
Removed.